config-docker.php

```diff
@@ -43,12 +43,12 @@
 /** Allow multiple short URLs for a same long URL
  ** Set to true to have only one pair of shortURL/longURL (default YOURLS behavior)
  ** Set to false to allow multiple short URLs pointing to the same long URL (bit.ly behavior) */
-define( 'YOURLS_UNIQUE_URLS', filter_var(getenv('YOURLS_UNIQUE_URLS'), FILTER_VALIDATE_BOOLEAN) ?: true );
+define( 'YOURLS_UNIQUE_URLS', getenv('YOURLS_UNIQUE_URLS') == 'false' ? false : true );
```

Comment:
OK for the way to go, but it can be linted to:

```suggestion
define( 'YOURLS_UNIQUE_URLS', getenv('YOURLS_UNIQUE_URLS') !== 'false' );
```

Reply:
I know. But I think in my version it is more clearly what the default value is.
I can change it, but people have to think twice (or more often) what will happen if the variable ist not set.

Reply:
I definitely prefer the shorten version.
All these variables are explained multiple times in documentation, and their names are sufficiently explicit to raise concerns.
People usually don't look at this file but the docs (don't forget this file is into distributed into a docker layer, not that easy to find and read).

Reply:
Ok, fixed.